### DISCUSSION TITLE
[resolved] Jak sprawdzić czy własne konto Allegro ma status Super Sprzedawcy?

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#4472] [Suggestions - resolved]
Przejrzałem wiele metod (w tym **/me, /sale/offers, /sale/user-ratings**) - ale nigdzie nie znalazłem informacji o statusie Super Sprzedawcy. Mówię tu oczywiście o własnych kontach zalogowanego użytkownika.
Paradoksalnie jedyna metoda, która zwraca taką informację to publicznie dostępna **/offers/listing** (bearer-token-for-app)
Jednakże ona pobiera listę aukcji, a przy każdej z nich zwraca pole **superSeller**: **true/false**.

Czy naprawdę żadna metoda z **Bearer-token-for-user** nie zwraca statusu super sprzedawcy???

**Comment by AureliuszBrussy** [comment]
Nie ma sposobu, aby sprawdzić, czy użytkownik jest Super Sprzedawcą - zwracamy taką informację tylko we wspomnianym przez Ciebie **GET /offers/listing**. Twoje zgłoszenie w takim przypadku potraktuję jako sugestię - zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przeniosłem wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions).

**Comment by AureliuszBrussy** [comment]
@javaguru-pl dodaliśmy już opcję sprawdzenia, czy sprzedający ma status Super Sprzedawcy - spojrz na [naszego newsa](https://github.com/allegro/allegro-api/issues/5184).

**Comment by daniel8702** [comment]
Cześć,

trochę stary już temat. Ale fajną opcją było by też dodanie informacji o poziomie procentowym czasowych odpowiedzi co ma 
ogromny wpływ na status SS, dzięki temu klienci mogli by kontrolować ten wskaźnik odpowiedzi. 
@AureliuszBrussy - czy była by szansa na taką relatywnie prostą funkcjonalność ? 

Pozdrawiam.

**Reply by Lukasz-Zurek** [reply]
@daniel8702 W imieniu Aureliusza mogę powiedzieć, że jest szansa na tego typu funkcjonalność. W [tym poście](https://github.com/allegro/allegro-api/discussions/5341#discussioncomment-2006164) napisałem, że planujemy wdrożenie funkcjonalności dotyczących jakości sprzedaży w naszym API. Prawdopodobnie już w przyszłym roku. Możesz oddać swój głos na sugestię opisaną w [tym wątku](https://github.com/allegro/allegro-api/discussions/5341#discussion-3814392).
